docs/contributor/04-30-ci-cd.md

```diff
@@ -7,61 +7,61 @@ The following CI jobs are part of the development cycle.
+| [`Pull Request integration tests / Kubernetes version compatibility test`](https://github.com/kyma-project/istio/blob/fd34803dd516c3b614fda4f5b9d9a4c8b46372bb/.github/workflows/pull-integration.yaml#L34) | Executes the basic integration test suite for the Istio module and checks if any incompatibilities occurred during the execution.                      |
+| [`Pull Request integration tests / Istio upgrade integration test`](https://github.com/kyma-project/istio/blob/fd34803dd516c3b614fda4f5b9d9a4c8b46372bb/.github/workflows/pull-integration.yaml#L45)        | Executes the upgrade integration test suite for the Istio module and verifies if the existing release can be successfully upgraded to the new version. |
+| [`Pull Request / lint`](https://github.com/kyma-project/istio/blob/fd34803dd516c3b614fda4f5b9d9a4c8b46372bb/.github/workflows/pull-request.yaml#L25)                                                        | Performs linting.                                                                                                                                      |
+| [`Pull Request / Run unit tests`](https://github.com/kyma-project/istio/blob/fd34803dd516c3b614fda4f5b9d9a4c8b46372bb/.github/workflows/pull-request.yaml#L40)                                              | Runs unit tests with code coverage information.                                                                                                        |
 
 ## Pipelines Running on the Release Branch
 
 The following CI jobs generate release artifacts and verify the functional correctness of Kyma Istio Operator.
 
 | Name                                                                                                                                                                         | Description                                                                             |
 |------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------|
-| [`Istio integration test`]() | Executes the main integration test suite for the Istio module on a k3d cluster.         |
-| [`rel-istio-build`]()           | Builds Kyma Istio Operator's image on the release and pushes it to the `prod` registry. |
+| [`Istio integration test`](https://github.com/kyma-project/istio/blob/fd34803dd516c3b614fda4f5b9d9a4c8b46372bb/.github/workflows/main-integration.yaml#L46) | Executes the main integration test suite for the Istio module on a k3d cluster.         |
```

Comment:
We also run the "main integration tests" workflow on release branches: https://github.com/kyma-project/istio/blob/d91645ab9c8935da03f82ed87623663cf2a621ad/.github/workflows/main-integration.yaml#L18
The question is also why we don't do it for api-gateway.

Reply:
I think we should do it in api-gateway

Reply:
I'll add it to my list and do it as a coffee break task once the documentation PR of api-gateway s merged.